build.gradle.kts

```diff
@@ -60,7 +60,6 @@ dependencies {
     testImplementation("com.h2database:h2")
     testImplementation("org.awaitility:awaitility")
     testImplementation("org.hamcrest:hamcrest-library")
-    testImplementation("org.scala-lang:scala-library:2.12.11")
```

Comment:
Fikk ikke testene til å kjøre før jeg fjernet denne, den var kanskje lagt inn for dette: https://stackoverflow.com/questions/64669007/noclassdeffounderror-scala-math-ordering-with-spring-kafka-test-2-5-7

Reply:
👍